core/domain/skill_domain.py

```diff
@@ -325,6 +325,71 @@ def validate(self):
                 self.explanation)
 
 
+class WorkedExample(python_utils.OBJECT):
+    """Domain object for representing the worked_example dict."""
+
+    def __init__(self, question, answer):
+        """Constructs a WorkedExample domain object.
+
+        Args:
+            question: SubtitledHtml. The example question.
+            answer: SubtitledHtml. The answer for the above example question.
+        """
+        self.question = question
+        self.answer = answer
```

Comment:
In the backend, can we call this "explanation" instead of "answer"?

Rationale: we already use "answer" for a typed object in the Solution class. This is more like the "explanation" since it includes how to get to the answer.

Reply:
Done